moveit_ros/manipulation/move_group_pick_place_capability/src/pick_place_action_capability.cpp

```diff
@@ -322,6 +322,7 @@ void move_group::MoveGroupPickPlaceAction::executePickupCallback(const moveit_ms
 {
   setPickupState(PLANNING);
 
+  context_->planning_scene_monitor_->waitForCurrentRobotState(ros::Time::now());
```

Comment:
Before we start planning, ensure that we have the latest robot state received...